src/tools/failurelogs-utils/automate.ts

```diff
@@ -64,16 +73,17 @@ export async function retrieveSessionFailures(
     },
   });
 
-  await assertOkResponse(response, "session logs");
+  const validationResult = validateLogResponse(response, "session logs");
```

Comment:
```suggestion
  const validationError = validateLogResponse(response, "session logs");
```
do this in all places